src/common/search/search-input.ts

```diff
@@ -22,6 +22,9 @@ class SearchInput extends LitElement {
   @property({ type: Boolean, attribute: "no-underline" })
   public noUnderline = false;
 
+  @property({ type: Boolean, attribute: "no-autofocus" })
+  public noAutofocus = false;
```

Comment:
Can we do this the other way around? Have an `autofocus` prop so it is aligned with the normal API?